packages/esm-patient-medications-app/src/add-drug-order/drug-order-form.component.tsx

```diff
@@ -812,7 +854,7 @@ const ControlledFieldInput = ({
       return (
         <NumberInput
           value={!!value ? value : 0}
-          onChange={(e, { value }) => handleChange(parseFloat(value))}
+          onChange={(e, { value }) => handleChange(isNaN(parseFloat(value)) ? null : parseFloat(value))}
```

Comment:
```suggestion
          onChange={(e, { value }) => {const number = parseFloat(value); handleChange(isNaN(number) ? null : number)} }
```

Reply:
The one thing I'm not seeing in this PR is if `requireOutpatientQuantity` is false, a `0` should be translated to `null`.